.github/scripts/check-filenames.js

```diff
@@ -0,0 +1,42 @@
+const fileNames = process.env.FILENAMES.split(' ');
+const baseFileNames = process.env.BASE_FILENAMES.split(' ');
+const user = process.env.USERNAME;
+let exitCode = 0;
+
+// Remove files not changed in this PR from list
+if (baseFileNames.length > 0) {
+  for(const file of baseFileNames) {
+    const index = fileNames.indexOf(file);

```

Comment:
Would using a `.filter` make this easier to read?

Reply:
Probably. I will work with it on my test repo & see how it goes.